stubs/config/queue.stub

```diff
@@ -1,11 +1,18 @@
-import Env from '@ioc:Adonis/Core/Env'
-import type { QueueConfig } from '@ioc:Rlanz/Queue'
+{{{
+  exports({ to: app.configPath('queue.ts') })
+}}}
+import env from '#start/env'
+import { defineConfig } from '@rlanz/bull-queue'
 
-const queueConfig: QueueConfig = {
+declare module 'Rlanz/Queue' {
+  interface JobsList {}
+}
```

Comment:
This currently results in a typescript error: `TS2664: Invalid module name in augmentation, module Rlanz/Queue cannot be found.` Is there a more appropriate place for this?